### ISSUE TITLE
Add ARM Builds

### ISSUE THREAD

**Issue by achow101** [bitcoin-core/HWI#345]
Seems like ARM builds might be something we should do: https://github.com/btcpayserver/BTCPayServer.Vault/issues/20

Which ARM platforms specifically should we build on, and what machines might be necessary? I've got Raspberry Pi 1, 3, and 4 so I think I can cover the most common ARM architectures.

**Comment by NicolasDorier** [comment]
In the wild I saw arm64 (say rock64pro) and arm32 (raspberry)

**Comment by NicolasDorier** [comment]
I think that it may interest wasabi wallet as well ping @nopara73 @lontivero @molnard

**Comment by molnard** [comment]
We decided to not support officially ARM platforms because of un-maintainability reasons.  .NET core >3 is capable to compile to it. @jmacato made a successful experiment with Wasabi and Pinebook PRO.

Sooner or later we have to deal with it according to the news, Apple is moving toward ARM but not now.
https://www.executivetraveller.com/news/apple-arm-macbook

**Comment by jmacato** [comment]
ARM64 will be more relevant in the coming years.. Currently Wasabi is working on arm64 platforms, I tested and I'm using it in my arm64 laptop and works okay. Though i had to make an out-of-band support for it.. for now at least.

**Comment by NicolasDorier** [comment]
Raspberry does not seem to want to transition to arm64 though.

**Comment by jmacato** [comment]
@NicolasDorier yeah... skia would still work there btw. We have some users that uses avalonia on RPi's.

**Comment by achow101** [comment]
It looks like AWS has ARM64 instances so I can use those to do ARM64 builds.

**Comment by KayBeSee** [comment]
I'd like to +1 on having ARM64 builds. Would be cool to be able to use hardware wallets on nodes like Umbrel.

**Comment by achow101** [comment]
Arm builds happened at some point
